external/upstream/libint/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-find_package(Libint 1.2.0 CONFIG QUIET COMPONENTS ${MAX_AM_ERI})
+find_package(Libint 1.2.1 CONFIG QUIET COMPONENTS ${MAX_AM_ERI})
```

Comment:
I acknowledge that I'm being inconsistent here. But every package _except_ Libint is cheap to build. And Libint hasn't changed materially (that is, other than build improvements) since 1.2.0. So I think move `find_package(Libint` back to 1.2.0, please. Do keep the `GIT_TAG` pointing to the new Windows compatible. All new builds should be latest version; just grandfather in the detecting old pre-built for Linux/Mac.

For reference, the MacOS took a week to build with AM6 and the Intel compilers with just two architecture optimizations.

Reply:
OK! I see you point, I will change back to `1.2.0`